### PR TITLE
feat: links

### PR DESCRIPTION
**Pull request by raghav6202**
Added links and signup components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 import logo from "./logo.png";
+import {   BrowserRouter as Router,Routes, Route } from "react-router-dom";
 import LandingPage  from "./pages/landing-page/landingPage.jsx";
 import  { Nav } from "./comps/navigation/navigation.jsx";
 import { ProductPage} from "./pages/product-page/productPage.jsx";
-import { CartPage } from "./pages/cart-page/cart-page.jsx"
+import { CartPage } from "./pages/cart-page/cart-page.jsx";
+import {LogInPage } from "./pages/authentication/login-page.jsx";
+import { WishList } from "./pages/wishlist-page/wishlist-page";
+import {SignUpPage} from "./pages/authentication/signup-page.jsx";
```
Comment (parulgupta2001) [inline src/App.js:10]: You should export all pages at one common file eg-index.js and then export

File: src/pages/authentication/authentication.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+@import url("https://jatayui.netlify.app/comp-lib.css");
+/* component library */
+
+@import url("./../landing-page/landingPage.css");
+
+.login-page-conatiner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0%;
```
Comment (parulgupta2001) [inline src/pages/authentication/authentication.css:12]: 0 is enough, you don't have to put any unit after 0

**Review by parulgupta2001 — APPROVED** [review]
apply the suggestion, otherwise everything looks good

**Review by sashaboi — APPROVED** [review]
Looks good to go !
I loved the theme and styling on the pages, looking forward to functionality of this project ahead

**Pull request merged**